helm/chart-operator/values.yaml

```diff
@@ -45,7 +45,6 @@ resource:
     name: '{{ .Release.Name | replace "." "-" | trunc 47 }}-psp'
 
 tiller:
-  image: "giantswarm/tiller:v2.16.8"
```

Comment:
This is the leftover from last Helm security incident. We need to remove this before rolling out a new release.

Reply:
Good catch, thanks!